package.json

```diff
@@ -2699,30 +2699,28 @@
     },
     "scripts": {
         "watch": "tsc -watch -p ./",
-        "build": "gulp build",
-        "ci-build": "gulp ci-build",
-        "package": "gulp package",
-        "ci-package": "gulp ci-package",
-        "lint": "gulp lint",
-        "test": "gulp test",
-        "test-only": "gulp test-only",
-        "unit-test": "gulp unit-test",
-        "webpack-dev": "gulp webpack-dev",
-        "webpack-prod": "gulp webpack-prod"
+        "build": "tsc -p ./",
+        "ci-build": "npm run test",
+        "package": "vsce package",
+        "ci-package": "npm run test && vsce package",
```

Comment:
This probably ends up running webpack twice

Reply:
If only there were a build orchestration tool that could avoid that... :-) Eh, the pendulum now at "pure scripts" will inevitably swing back the other way again.

Reply:
If only gulp weren't dead :smile: